workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -172,42 +173,43 @@ jobs:
       - name: Unzip release asset
         run: unzip -qq ${{ env.ASSET_ZIP }} -d ./${{ env.UNZIPPED_ASSET }}
 
-      - name: Set App Configuration and Substitution Values
+      - name: Set Variables for use with Configuration and Substitution Values
         uses: im-open/set-environment-variables-by-scope@v1.0.2
         with:
           scope: ${{ needs.set-vars.outputs.ENVIRONMENT }}
           create-output-variables: true
           input-file: '' # TODO:  if you have an input file with your variable substitutions, add them here
         env:
-          # TODO:  Add the values your app needs to be configured with (appsettings.json/octo var substitution/config changes). Envs that aren't used can be deleted.
-          #  - The name of the env (left side of @) (ex: Auth0_Domain).  This can be used in the later variable-substitution and octostache actions for replacement
-          #  - The scope (right side of @) should be dev/qa/stage/uat/demo/prod. This action uses set-vars.ENVIRONMENT to match one of the scopes below and these are the values it will have.
+          # TODO:  Add the values your app/function needs to be configured with (appsettings.json/local.settings.json/octo var substitution/config changes).  Delete envs that are not used.
+          #  - The name of the env (left side of @) (ex: Auth0_Domain) should be added.  The env name can be used in the later variable-substitution and octostache actions for replacement.
+          #  - The scope (right side of @) should be dev/qa/stage/uat/demo/prod. This action uses the value of set-vars.ENVIRONMENT to match one of the scopes (envs) below.
```

Comment:
This is sort of a "while we're in here" kind of thing, but do we want to mention you can do multiple environments in one declaration, e.g. `Auth0_Domain@dev qa: 'wtw-im-dev-auth0.com'`?

Reply:
Yeah I can do that, stage is actually a perfect example because demo uses it as well.